src/components/app-configure/ConfigureSidebar.vue

```diff
@@ -267,24 +303,26 @@ watch(
           </div>
           <div
             v-if="tab.subMenu && showConfigureSubmenu"
-            class="sidebar__submenu"
+            class="sidebar__submenu | space-y-3"
           >
-            <VCardButton
-              v-for="subTab in tab.subMenu"
-              :key="subTab.label"
-              class="sidebar__submenu-option"
-              :class="{ 'submenu-active': props.currentTab === subTab.label }"
-              @click.stop="onClickOfMenu(subTab)"
-            >
-              <div class="sidebar__submenu-option-item">
-                <img
-                  :src="subTab.icon"
-                  alt="icon"
-                  class="sidebar__submenu-option-icon"
-                />
-                <span class="submenu-tab-label">{{ subTab.label }}</span>
-              </div>
-            </VCardButton>
+            <div v-for="subTab in tab.subMenu" :key="subTab.label">
+              <VCardButton
+                class="sidebar__submenu-option"
+                :class="{ 'submenu-active': isSubmenuSelected(subTab.label) }"
+                @click.stop="onClickOfMenu(subTab)"
+              >
+                <div class="space-y-3 flex flex-col">
```

Comment:
We can use `gap-3` instead of `space-y-3` wherever we are using `flex` and `grid`

Reply:
[done](https://github.com/arcana-network/developer-dashboard/pull/295/commits/0d69c93b4cb98728da38188bdb528894405157d3)